src/AppBundle/Entity/VoteSettings.php

```diff
@@ -0,0 +1,180 @@
+<?php
+
+namespace AppBundle\Entity;
+
+use Doctrine\Common\Collections\ArrayCollection;
+use Doctrine\ORM\Mapping as ORM;
+use Symfony\Component\Validator\Constraints as Assert;
+
+/**
+ * VoteSettings
+ *
+ * @ORM\Table()
+ * @ORM\Entity(repositoryClass="AppBundle\Entity\VoteSettingsRepository")
+ */
+class VoteSettings
+{
+    use GedmoTrait;
+
+    /**
+     * @var integer
+     *
+     * @ORM\Column(type="integer")
+     * @ORM\Id
+     * @ORM\GeneratedValue(strategy="AUTO")
+     */
+    private $id;
+
+    /**
+     * @var string
+     *
+     * @ORM\Column(type="string", length=255, nullable=true)
+     */
+    private $title;
+
+    /**
+     * @var integer
+     *
+     * @ORM\Column(name="vote_limits", type="integer", nullable=false)
+     */
+    private $voteLimits;
+
+    /**
+     * @var Project[]ArrayCollection
+     *
+     * @ORM\OneToMany(targetEntity="Project", mappedBy="voteSetting", cascade={"persist"})
+     */
+    private $project;
+
+    /**
+     * @var Location
+     *
+     * @ORM\ManyToOne(targetEntity="Location", inversedBy="voteSetting")
+     * @ORM\JoinColumn(name="location_id", nullable = true, referencedColumnName="id")
+     */
+    private $location;
```

Comment:
we have test case about this - manage vote setting for different vote in deferent cities and I create my realization, for ony city can be many vote setting, with count vote, with date from/to and type project (I add type in weekend) I think this work be. If yoy have another realization you can craete new branch and welcam